test/io/Jamfile

```diff
@@ -0,0 +1,7 @@
+import testing ;
+
+run t_hdu.cpp;
```

Comment:
Add space between the name of file and semicolon like
`run t_hdu.cpp ;`
for all.